package.json

```diff
@@ -15,7 +15,7 @@
     "cypress:test": "lerna run cypress:test",
     "commit": "git-cz",
     "bump:version": "node ./scripts/deploy/bump-version.js",
-    "npm:publish:alpha": "lerna run --no-bail npm:publish -- -- --tag alpha",
+    "npm:publish:alpha": "lerna run npm:publish -- -- --tag alpha --verbose --no-verify",
```

Comment:
I'd prefer if the `lerna` arguments were place after `run` instead of at the end of the line, to distinguish from the `--tag` argument which is passed to `npm publish`.

Is removing `--no-bail` necessary? If not, would that avoid needing to introduce a nodejs script?
Also, what does `--no-verify` do?

Reply:
My bad, I didn't check if `--no-verify` exists, I thought it was a npm argument.

It appears that Olivier meant `--no-verify-access`, which only exists for `lerna publish`.